Update frontend README with more info on running the perf tests (#5410)
kiali/kiali: frontend/cypress/README.md

## Performance Tests

> These coarsly measure metrics such as page load time. These are meant to give a general baseline of performance but are not useful for benchmarking. Here's roughly how these tests are expected to be used:

Reviewer: ```suggestion
These tests coarsely measure metrics such as page load time. They are meant to give a general baseline of performance but are not useful for benchmarking.  This is roughly how to use the tests:
```

yarn cypress --config-file cypress-perf.json
```

### Parameterizing tests

You can adjust some inputs of the performance tests by changing the [fixture files](fixtures/perf/).

> ### Limiting suite to a single test

Reviewer: ```suggestion
### Limiting the suite to a single test
```
